modules/ting/ting.entities.inc

```diff
@@ -380,23 +380,25 @@ class TingCollection extends DingEntityCollection {
     parent::__construct();
   }
 
-  function getTitle() {
-    foreach ($this->reply->objects as &$object) {
+  /**
+   * The title of the collection.
+   *
+   * @return string|FALSE
+   *   The title or FALSE if it could not be determined.
+   */
+  public function getTitle() {
+    foreach ($this->tingObjectCollection->getObjects() as $object) {
       // Find the title of the object that was used to fetch this collection. We
       // don't look at dkdcplus:full as it might be too specific ("1Q84. Book 1"
       // for instance).
-      if ($object->id == $this->ding_entity_id && !empty($object->record['dc:title'])) {
-        return $object->record['dc:title'][''][0];
+      if ($object->getId() === $this->getId() && !empty($object->getShortTitle())) {
+        return $object->getShortTitle();
```

Comment:
The TingObject interface mentions something about a fallback from short of default title if short is not available. That does not seem to be the case here?

Reply:
I'm following the intent of the original code. The comment above the line says that the full title might be to specific (ie. the books first title might reveal that the title is actually not a collection-title but just the title of the first book). So that is why we always use the short title for collections.

Reply:
I thought so - I just wondered if the interface comment was actually true.